proto/p4/config/p4info.proto

```diff
@@ -80,26 +78,29 @@ message Table {
   repeated int32 action_ids = 3;
   // 0 (default value) means that the table does not have a const default action
   int32 const_default_action_id = 4;
```

Comment:
suggest s/int32/uint32/ for ID type (see my comment above)
[Here and elsewhere]